### ISSUE TITLE
`litestream restore`: no valid providers in chain

### ISSUE THREAD

**Issue by WesleyAC** [benbjohnson/litestream#260]
With a config file like:

```yaml
access-key-id: REDACTED
secret-access-key: REDACTED

dbs:
  - path: /home/username/dbname.sqlite3
    replicas:
      - url: s3://some-digitalocean-space.nyc3.digitaloceanspaces.com/dbname/
```

The command `litestream restore /home/username/dbname.sqlite3` works fine, whereas `litestream restore -o dbname.sqlite3 s3://some-digitalocean-space.nyc3.digitaloceanspaces.com/dbname/` gives the following error:

```
cannot fetch generations: NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
```

This seems like a bug to me, based on my understanding of the [`litestream restore` docs](https://litestream.io/reference/restore/), which give an example of doing basically what I'm trying to do (`litestream restore -o /tmp/my.db s3://mybkt.litestream.io/my.db
`).

`litestream version` reports `v0.3.7`, I'm using the deb package.

**Comment by benbjohnson** [comment]
@WesleyAC Yes, using a URL with `litestream` commands skips the config file because most of it doesn't apply. However, it should at least pull in the global access key & secret key. I tagged this for fix in v0.4.0.

**Comment by benbjohnson** [comment]
This is working in `master`
